tomcat/src/main/java/nextstep/jwp/db/InMemoryUserRepository.java

```diff
@@ -23,5 +28,16 @@ public static Optional<User> findByAccount(String account) {
         return Optional.ofNullable(database.get(account));
     }
 
+    public static User findByAccountAndPassword(String account, String password) {
+        User user = Optional.ofNullable(database.get(account))
+                .orElseThrow(InvalidLoginException::new);
```

Comment:
이 코드는 중복입니다. 살린다면 이렇게 쓰는 건 어떨까요?
```suggestion
        User user = findByAccount(account).orElseThrow(() -> new InvalidLoginException));
```

Reply:
앗 그렇네요 감사합니다!!